toolchain/driver/testdata/error_codegen_objcode_output_file_without_triple.carbon

```diff
@@ -0,0 +1,8 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
```

Comment:
The name of this test file doesn't match what it's doing: it says `output_file_without_triple`, but it has neither an output file nor a target specified.